src/main/java/com/powsybl/openloadflow/sa/AcSecurityAnalysis.java

```diff
@@ -135,13 +189,31 @@ private SecurityAnalysisResult runSimulations(LfNetwork network, List<Propagated
             network.setReporter(preContSimReporter);
 
             // run pre-contingency simulation
-            AcLoadFlowResult preContingencyLoadFlowResult = new AcloadFlowEngine(context)
-                    .run();
+            // FIXME
```

Comment:
@geofjamg here is ugly too because I need the equation system and the listener to be created in order to listen to the `setDisabled` I am doing on switches to close.